dist/win/resources/main.wxs

```diff
@@ -86,12 +86,12 @@
     <!-- Non-Opening ProgID -->
     <DirectoryRef Id="INSTALLDIR">
         <Component Win64="yes" Id="nonStartingProgID" >
-          <File Id="IconFileForEncryptedData" KeyPath="yes" Source="$(env.JP_WIXWIZARD_RESOURCES)\Cryptomator-Vault.ico" Name="Cryptomator-Vault.ico"></File>
-          <ProgId Id="Cryptomator.Encrypted.1" Description="Cryptomator Encrypted Data" Icon="IconFileForEncryptedData" IconIndex="0">
-            <Extension Id="c9r" Advertise="no" ContentType="application/vnd.cryptomator.encrypted">
-              <MIME ContentType="application/vnd.cryptomator.encrypted" Default="yes"></MIME>
+          <File Id="IconFileForEncryptedData" KeyPath="yes" Source="$(env.JP_WIXWIZARD_RESOURCES)\$(var.JpAppName)-Vault.ico" Name="$(var.JpAppName)-Vault.ico"></File>
```

Comment:
Should be a new variable, e.g.

```suggestion
          <File Id="IconFileForEncryptedData" KeyPath="yes" Source="$(env.JP_WIXWIZARD_RESOURCES)\$(var.IconFileC9r)" Name="$(var.IconFileC9r)"></File>
```

Reply:
Resolved in [7dfc4d9](https://github.com/cryptomator/cryptomator/pull/2242/commits/7dfc4d9ded30d233e5d06e359dd96eccdf84c92c)